checks/E1F2C3.yaml

```diff
@@ -0,0 +1,79 @@
+id: "E1F2C3"
+name: SAPHanaController roles correctly configured
+group: Pacemaker
+description: |
+  SAPHanaController roles correctly configured
+remediation: |
+  ## Abstract
+  SAPHanaController roles should be configured as "Promoted" and "Unpromoted" and not as "Master" and "Slave".
+
+  ## Remediation
+
+  Edit the crm configuration and change the values of roles as per the best parctices.
+
+  ## References
+  AZURE:
```

Comment:
We have `AZURE`, `azure`, space after it, no space (the same for all providers)
Well, up to you folks. I'm just pointing out

Reply:
Thank you @arbulu89 ,, I see that there are more "Azure" than "AZURE". Maybe because Azure is not an acronym like GCP and AWS. Therefore I am going to change it to "Azure" for these checks. I am going to put one line space between the provider and the references, also as seen in most of the existing checks.